src/test/ui/packed/packed-struct-borrow-element.rs

```diff
@@ -1,4 +1,5 @@
 // run-pass (note: this is spec-UB, but it works for now)
+// ignore-32bit (needs `usize` to be 8-aligned to reproduce all the errors below)
```

Comment:
@petrochenkov I set this test to be ignored on 32bit, since it shows fewer errors there (`usize` in a `packed(4)` struct is actually okay on 32bit). Is that okay?

Reply:
Maybe it's better to change `usize` to something more predictable instead?

Reply:
I don't know any type that has alignment >4 on 32bit platforms, and using `#[repr(align(8))]` doesn't work as such types cannot be put into packed structs.

Reply:
Then maybe this case can be put under a revision.

Reply:
I don't know how that would work. Can I make a revision "32-bit-only"? Is that a thing?

Reply:
I tried
```
// revisions: all only64bit
//[only64bit] ignore-32bit (needs `usize` to be 8-aligned to reproduce all the errors below)
```
But that does not seem to work, the "only64bit" revision is still run on 32-bit systems (and hence causes a mismatch in the stderr file).

Reply:
I ended up just making this two separate tests.

@rustbot flags -S-waiting-on-author +S-waiting-on-review